me.kozec.syncthingtk.yaml

```diff
@@ -41,27 +41,46 @@ modules:
       - /share/man
 
   - name: syncthing
+    disabled: false
```

Comment:
No need for this.

Reply:
maybe. But it significantly reduces the overhead for packaging newer versions of the desktop app. The git dependencies are very slow to load with flatpak-builder.